generation/tasks/RunRivet.py

```diff
@@ -154,25 +152,23 @@ def output(self):
 
     def run(self):
         # branch data
+        _map = {
```

Comment:
Again this mapping needs to be generalized in order to work for other processes as well.

Reply:
I will make a new PR for the new and improve RivetMerge. This is out of scope for this PR.